### ISSUE TITLE
TypeError: 'module' object is not callable

### ISSUE THREAD

**Issue by dvzrv** [owncloud/news-updater#4]
I think the commit d5af6f6a50f68825cd4648a41bb301a449ed363b broke the way the script that'll be placed in /usr/bin/owncloud-news-updater to correctly call the rest of the code.

I am now getting this, when calling the script:

```
owncloud-news-updater[21508]: Traceback (most recent call last):
owncloud-news-updater[21508]:   File "/usr/bin/owncloud-news-updater", line 9, in <module>
owncloud-news-updater[21508]:     load_entry_point('owncloud-news-updater==9.0.0', 'console_scripts', 'owncloud-news-updater')()
owncloud-news-updater[21508]: TypeError: 'module' object is not callable
```

**Comment by BernhardPosselt** [comment]
Hm, works for me. Can you try to:

```
rm /usr/bin/owncloud-news-updater
```

and 

```
sudo python3 setup.py install --install-scripts=/usr/bin
owncloud-news-updater -v
```

**Comment by dvzrv** [comment]
That produces this:

```
running install
running bdist_egg
running egg_info
writing entry points to owncloud_news_updater.egg-info/entry_points.txt
writing dependency_links to owncloud_news_updater.egg-info/dependency_links.txt
writing top-level names to owncloud_news_updater.egg-info/top_level.txt
writing owncloud_news_updater.egg-info/PKG-INFO
reading manifest file 'owncloud_news_updater.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'owncloud_news_updater.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/owncloud_news_updater
creating build/bdist.linux-x86_64/egg/owncloud_news_updater/common
copying build/lib/owncloud_news_updater/common/logger.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/common
copying build/lib/owncloud_news_updater/common/argumentparser.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/common
copying build/lib/owncloud_news_updater/common/__init__.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/common
creating build/bdist.linux-x86_64/egg/owncloud_news_updater/api
copying build/lib/owncloud_news_updater/api/api.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/api
copying build/lib/owncloud_news_updater/api/updater.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/api
copying build/lib/owncloud_news_updater/api/cli.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/api
copying build/lib/owncloud_news_updater/api/web.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/api
copying build/lib/owncloud_news_updater/api/__init__.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/api
copying build/lib/owncloud_news_updater/container.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater
copying build/lib/owncloud_news_updater/version.txt -> build/bdist.linux-x86_64/egg/owncloud_news_updater
copying build/lib/owncloud_news_updater/__main__.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater
copying build/lib/owncloud_news_updater/version.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater
creating build/bdist.linux-x86_64/egg/owncloud_news_updater/dependencyinjection
copying build/lib/owncloud_news_updater/dependencyinjection/container.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/dependencyinjection
copying build/lib/owncloud_news_updater/dependencyinjection/__init__.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater/dependencyinjection
copying build/lib/owncloud_news_updater/__init__.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater
copying build/lib/owncloud_news_updater/config.py -> build/bdist.linux-x86_64/egg/owncloud_news_updater
creating build/bdist.linux-x86_64/egg/tests
creating build/bdist.linux-x86_64/egg/tests/owncloud_news_updater
creating build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api
copying build/lib/tests/owncloud_news_updater/api/test_web.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api
copying build/lib/tests/owncloud_news_updater/api/test_cli.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api
copying build/lib/tests/owncloud_news_updater/api/__init__.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api
creating build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/updaters
copying build/lib/tests/owncloud_news_updater/updaters/__init__.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/updaters
copying build/lib/tests/owncloud_news_updater/test_config.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater
creating build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/dependencyinjection
copying build/lib/tests/owncloud_news_updater/dependencyinjection/test_container.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/dependencyinjection
copying build/lib/tests/owncloud_news_updater/dependencyinjection/__init__.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/dependencyinjection
copying build/lib/tests/owncloud_news_updater/__init__.py -> build/bdist.linux-x86_64/egg/tests/owncloud_news_updater
copying build/lib/tests/__init__.py -> build/bdist.linux-x86_64/egg/tests
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/common/logger.py to logger.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/common/argumentparser.py to argumentparser.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/common/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/api/api.py to api.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/api/updater.py to updater.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/api/cli.py to cli.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/api/web.py to web.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/api/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/container.py to container.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/__main__.py to __main__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/version.py to version.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/dependencyinjection/container.py to container.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/dependencyinjection/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/owncloud_news_updater/config.py to config.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api/test_web.py to test_web.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api/test_cli.py to test_cli.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/api/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/updaters/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/test_config.py to test_config.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/dependencyinjection/test_container.py to test_container.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/dependencyinjection/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/owncloud_news_updater/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/tests/__init__.py to __init__.cpython-35.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying owncloud_news_updater.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying owncloud_news_updater.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying owncloud_news_updater.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying owncloud_news_updater.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying owncloud_news_updater.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
owncloud_news_updater.__pycache__.version.cpython-35: module references __file__
tests.owncloud_news_updater.__pycache__.__init__.cpython-35: module references __file__
creating dist
creating 'dist/owncloud_news_updater-9.0.0-py3.5.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing owncloud_news_updater-9.0.0-py3.5.egg
creating /usr/lib/python3.5/site-packages/owncloud_news_updater-9.0.0-py3.5.egg
Extracting owncloud_news_updater-9.0.0-py3.5.egg to /usr/lib/python3.5/site-packages
Adding owncloud-news-updater 9.0.0 to easy-install.pth file
Installing owncloud-news-updater script to /usr/bin

Installed /usr/lib/python3.5/site-packages/owncloud_news_updater-9.0.0-py3.5.egg
Processing dependencies for owncloud-news-updater==9.0.0
Finished processing dependencies for owncloud-news-updater==9.0.0
9.0.0
```

And doesn't change the error message unfortunately ;-)

**Comment by BernhardPosselt** [comment]
Ok, asked around in #python on freenode and apparently you need to use a function, https://github.com/owncloud/news-updater/commit/6dbf2be01ae66060daec155ee9047d3cc3ac14fb should fix that, can you try the latest master?

No idea why it worked here btw :)

**Comment by dvzrv** [comment]
I can confirm, that this now works.
Funny, I was suspecting that exact thing!

**Comment by BernhardPosselt** [comment]
Released 9.0.1 this should be fixed now :)
